server/src/engineModel/EngineModel.ts

```diff
@@ -227,8 +238,12 @@ export class EngineModel {
     private importTags(dataAndFeatures: any): void {
         // Get the single “text-tags” element.
         const textTags = dataAndFeatures['text-tags'][0];
-        textTags.tag.forEach((tag) => {
-            const textTag = this.buildYmlFunction(tag);
+        textTags.tag.forEach((_tag) => {
+            if (!_tag.$.ident) {
```

Comment:
Can’t remember what those tags are and what is happening here :| sorry.
Could you please refresh my memory with examples or explanations or whatever?

Reply:
Tags in YML are the elements that are like functions, but starting with `\`. They are used in `TextGranule` and `TextFunction` objects.

Reply:
And what about `.$.ident`?

Reply:
`.$` is the object containing the tag's attributes. `ident` is one of them (the YML identifier).

Reply:
How can I test that stuff?